OpenMTRDemo/Forms/TiledFiltersForm.cs

```diff
@@ -0,0 +1,391 @@
+            DemoUtilities.loadImage(pictureBoxPane4, meterPane4.SourceImage);
+            DemoUtilities.loadImage(pictureBoxPane5, meterPane5.SourceImage);
+            DemoUtilities.loadImage(pictureBoxPane6, meterPane6.SourceImage);
+            DemoUtilities.loadImage(pictureBoxGray, grayMat);
+            DemoUtilities.loadImage(pictureBoxSobel, sobelMat);
+            DemoUtilities.loadImage(pictureBoxCanny, cannyMat);
+            DemoUtilities.loadImage(pictureBoxLaplace, laplacianMat);
+            DemoUtilities.loadImage(pictureBoxScharr, scharrMat);
+        }
+
+        private void pictureBoxSource_DoubleClick(object sender, EventArgs e)
+        {
+            ExpandedImageForm expandedImageForm = new ExpandedImageForm(meter.SourceImage);
+            expandedImageForm.ShowDialog();
+            expandedImageForm.Close();
+        }
+
+        private void pictureBoxGray_DoubleClick(object sender, EventArgs e)
+        {
+            ExpandedImageForm expandedImageForm = new ExpandedImageForm(grayMat);
+            expandedImageForm.ShowDialog();
+            expandedImageForm.Close();
+        }
+
+        private void pictureBoxCanny_DoubleClick(object sender, EventArgs e)
+        {
+            ExpandedImageForm expandedImageForm = new ExpandedImageForm(cannyMat);
+            expandedImageForm.ShowDialog();
+            expandedImageForm.Close();
+        }
+
+        private void pictureBoxSobel_DoubleClick(object sender, EventArgs e)
+        {
+            ExpandedImageForm expandedImageForm = new ExpandedImageForm(sobelMat);
+            expandedImageForm.ShowDialog();
+            expandedImageForm.Close();
+        }
+
+        private void pictureBoxLaplace_DoubleClick(object sender, EventArgs e)
+        {
+            ExpandedImageForm expandedImageForm = new ExpandedImageForm(laplacianMat);
+            expandedImageForm.ShowDialog();
+            expandedImageForm.Close();
+        }
+
+        private void pictureBoxScharr_DoubleClick(object sender, EventArgs e)
+        {
+            ExpandedImageForm expandedImageForm = new ExpandedImageForm(scharrMat);
+            expandedImageForm.ShowDialog();
+            expandedImageForm.Close();
+        }
+
+        private void pictureBoxPane1_DoubleClick(object sender, EventArgs e)
+        {
+            try
+            {
+                ExpandedImageForm expandedImageForm;
+                if (meterPane1.ModifiedImage == meterPane1.SourceImage)
+                {
+                    expandedImageForm = new ExpandedImageForm(meterPane1.SourceImage);
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
+                else
+                {
+                    expandedImageForm = new ExpandedImageForm(meterPane1.ModifiedImage);
+                    expandedImageForm.ShowDialog();
+                    expandedImageForm.Close();
+                }
```

Comment:
This would be more efficient here
``` C#
if (meterPane1.ModifiedImage == meterPane1.SourceImage)
{
    expandedImageForm = new ExpandedImageForm(meterPane1.SourceImage);
}
else
{
    expandedImageForm = new ExpandedImageForm(meterPane1.ModifiedImage);
}
expandedImageForm.ShowDialog();
expandedImageForm.Close();
```
However I don't actually see a reason to have this if statement at all, because expandedImageForm winds up being equal to ModifiedImage in both cases.

Reply:
☝️ I agree. 

```C#
expandedImageForm = new ExpandedImageForm(meterPane1.ModifiedImage);
expandedImageForm.ShowDialog();
expandedImageForm.Close();
```